packages/design/src/Borders.mdx

```diff
@@ -0,0 +1,64 @@
+---
+name: Dividing Content
+menu: Design
+route: /borders
+---
+
+# Borders and Dividers
+
+You may find that a visual indicator is helpful to segment elements in an
+interface. We have elements to help you do so in a consistent way across
+platforms: borders and dividers.
+
+## Borders
+
+### Border Widths
+
+To add a border to an element, use the border-width variables in the component's
+styling to ensure consistent border thickness throughout Jobber.
+
+Border style should be `solid` unless a specific use case calls for some other
+format, such as indicating a dropzone on the
+[`InputFile` component](https://atlantis.getjobber.com/components/input-file).
+
+export function Example({ of: size }) {
```

Comment:
can we name this something more descriptive.

Reply:
Variations of this component is actually used in a few different design guides. I wonder if we can generalize them. 🤔

Reply:
I think I'll leave this as is for now, and we can see if there's a good standard... I think if I leave this as `Example` and we come back to standardizing, it'll be easier to flag at that point?